.storybook/blocks/ComponentDetails.jsx

```diff
@@ -119,6 +157,175 @@ const VersionDetails = ({ tag, data = {}, isDeprecated = false, skipDate = false
 	);
 };
 
+// link icons/svgs for ResourceLinkContent
+const npmSvg = () => {
+	return (
+		<ResourceIconWrapper backgroundColor="rgba(203, 56, 55, 0.1)">
+			<svg viewBox="0 0 18 7" focusable="false" aria-hidden="true" aria-label="npm">
+				<path fill="#CB3837" d="M0,0h18v6H9v1H5V6H0V0z M1,5h2V2h1v3h1V1H1V5z M6,1v5h2V5h2V1H6z M8,2h1v2H8V2z M11,1v4h2V2h1v3h1V2h1v3h1V1H11z"></path>
+				<polygon fill="#FFFFFF" points="1,5 3,5 3,2 4,2 4,5 5,5 5,1 1,1 "></polygon>
+				<path fill="#FFFFFF" d="M6,1v5h2V5h2V1H6z M9,4H8V2h1V4z"></path>
+				<polygon fill="#FFFFFF" points="11,1 11,5 13,5 13,2 14,2 14,5 15,5 15,2 16,2 16,5 17,5 17,1 "></polygon>
+			</svg>
+		</ResourceIconWrapper>
+	)
+};
```

Comment:
I am super open to feedback on whether or not all of the SVG-related code here makes sense or could be refactored.